src/Spice86.Shared/Emulator/Memory/SegmentedAddress.cs

```diff
@@ -34,17 +17,40 @@ public SegmentedAddress(ValueTuple<ushort, ushort> segmentOffset) : this(segment
     public SegmentedAddress(ushort segment, ushort offset) {
         Segment = segment;
         Offset = offset;
+        Linear = (uint)(Segment << 4) + Offset;
```

Comment:
Now SegmentedAddress fits in 64bits instead of 32.
This should still not have any negative impact on performance because the value can still be transported via one register, but how is the performance compared to before the change?

Reply:
No measruable difference, also with cfgcpu